dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/All.java

```diff
@@ -72,13 +72,13 @@ public void execute(final Namespace ns, final DockerfileGitHubUtil dockerfileGit
                 changeDockerfiles(ns, pathToDockerfilesInParentRepo, imagesFoundInParentRepo, imageToTagMap, currUserRepo,
                         skippedRepos);
             } catch (IOException e) {
+                log.error(String.format("Error changing Dockerfile for %s", currUserRepo.getName()), e);
```

Comment:
Seems advantageous to go ahead and log this immediately to be able to envision flow a bit.